packages/workspace/src/utils/buildable-libs-utils.ts

```diff
@@ -1,315 +0,0 @@
-import { ProjectType } from '../core/project-graph';
```

Comment:
We should keep this file and the above file for backwards compatibility. Some people might still import from this path even though it's a deep import.

We can remove them in Nx v13.

Reply:
Ok, I'll mark those functions then as deprecated